torch/autograd/__init__.py

```diff
@@ -250,9 +251,10 @@ def grad(
         create_graph (bool, optional): If ``True``, graph of the derivative will
             be constructed, allowing to compute higher order derivative products.
             Default: ``False``.
-        allow_unused (bool, optional): If ``False``, specifying inputs that were not
-            used when computing outputs (and therefore their grad is always zero)
-            is an error. Defaults to ``False``.
+        allow_unused (Optional[bool], optional): If ``False``, specifying inputs 
+            that were not used when computing outputs (and therefore their grad is 
+            always zero) is an error. If ``None``, it will follow the value of 
```

Comment:
Maybe we could simplify the wording of this and say: `Defaults to the value of ``materialize_grad`` `